### ISSUE TITLE
RAF will not import via CLI

### ISSUE THREAD

**Issue by Thoroslives** [immich-app/CLI#79]
CLI will not import RAF files.
Its a known issue
Seeking to get it fixed as i have a huge library of RAF that would be impractical to upload via webui

**Comment by Thoroslives** [comment]
Is this issue section active?

**Comment by bo0tzz** [comment]
Yes, but most of the attention is going to the main Immich project at the moment.

**Comment by Thoroslives** [comment]
> Yes, but most of the attention is going to the main Immich project at the moment.

Okay thats fair enough, i suppose through the web ui is the only option then
